JsonLow.d.ts

```diff
@@ -40,7 +115,7 @@ export type JsonLowHandlers<Feedback, End> = {
   comma?: JsonLowHandler<Feedback>,
   colon?: JsonLowHandler<Feedback>,
 
-  state?: () => string,
+  state?: () => JsonLowState,
```

Comment:
(see the previous comment about the `downstream` prop)

This should have the same type as `downstream` -- either generic `T` (or a more descriptive name) or `unknown`.

Reply:
Ditto